### PR TITLE
`fetchStatus` for deribit

### PR DESCRIPTION
**Pull request by ttodua**

### CODE REVIEW

**Comment by ttodua** [comment]
@kroitor  travis fails on this PR and on #12966 too. on local, it builds well.

File: js/deribit.js [hunk 1]
```diff
@@ -388,26 +388,28 @@ module.exports = class deribit extends Exchange {
...
+        //         "usOut": 1650641690226836,
+        //         "usDiff": 48,
+        //         "testnet": false
         //     }
         //
-        this.status = this.extend (this.status, {
-            'status': 'ok',
-            'updated': this.milliseconds (),
+        const result = this.safeString (response, 'result');
+        const locked = this.safeString (result, 'result') !== 'false';
```
Comment (kroitor) [inline js/deribit.js:405]: This line is not transpileable – break it down into multiple lines.

File: js/deribit.js [hunk 2]
```diff
@@ -388,26 +388,28 @@ module.exports = class deribit extends Exchange {
...
+        const updateTime = this.safeIntegerProduct (response, 'usIn', 0.001, this.milliseconds ());
+        return {
+            'status': (locked ? 'maintenance' : 'ok'),
```
Comment (kroitor) [inline js/deribit.js:408]: Should be `'status': locked ? 'maintenance' : 'ok',`

File: js/deribit.js (hunk 1 above)
Comment (ttodua) [reply js/deribit.js]: i've changed to `===` instead of `!==`, it should transpile now.

**Pull request merged**